packages/components/src/reactions_row/index.jsx

```diff
@@ -11,12 +11,13 @@ const ReactionsRow = ({
   reactions,
   onClick,
   className: passedClassName,
+  showNull = true,
   ...cleanProps
 }) => {
   const { list: sortedReactionTypes, count } = useReactionData(reactions);
   const reactionTypes = sortedReactionTypes.slice(-limit);
 
-  if (withCounter && count === 0) return null;
+  if (withCounter && count === 0) return showNull ? null : 0;
```

Comment:
won't work so easy because you still need to apply all the passed stuff like `onClick`, `passedClassName`